docs/api/remote-procedure-calls-evo.md

```diff
@@ -1534,9 +1534,9 @@ Result:
 }
 ```
 
-### ProTx Update Service HPMN
+### ProTx Update Service EvoNode
 
-The "protx update_service_hpmn" RPC creates and sends a ProUpServTx to the network. This will update the IP address and the Platform fields of a HPMN. If this is done for a HPMN that got PoSe-banned, the ProUpServTx will also revive this HPMN.
+The "protx update_service_hpmn" RPC creates and sends a ProUpServTx to the network. This will update the IP address and the Platform fields of a EvoNode. If this is done for a EvoNode that got PoSe-banned, the ProUpServTx will also revive this EvoNode.
```

Comment:
```suggestion
The "protx update_service_hpmn" RPC creates and sends a ProUpServTx to the network. This will update the IP address and the Platform fields of an evonode. If this is done for an evonode that was PoSe-banned, the ProUpServTx will also revive this evonode.
```